CONTRIBUTING.md

```diff
@@ -90,41 +90,43 @@ ID	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	SIMILARITY
+3 Have you added your reference for the concept list, and all references which you used in your description, to our bibtex file [```references.bib```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/references/references.bib)?
 4 Have you added new concept sets to our Concepticon in the file [```concepticon.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv)? If so, have you offered proper definitions?
-5 Do you reference a source file in PDF, in your description of the concept list in [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv)? If so, have you added it (in the form ```FirstAuthorLastNameYear```) to the folder [```sources/```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/sources/)?
-6 Do you have to thank some person for help with the concept list? If you are contributing and not part of our core editorial team, we will add you, so please provide us with your name and affiliation. To keep track of whom to thank, please edit the [```CONTRIBUTORS.MD```](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTORS.md) document.
+5 Do you reference a source file in PDF, in your description of the concept list in [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv)? If so, have you added it (in the form ```FirstAuthorLastNameYear```) to the folder [```sources/```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/sources/)? (AT: Might also need editing.)
+6 Do you have to thank a person for help with the concept list? If you are contributing and not part of our core editorial team, we will add you, so please provide us with your name and affiliation. To keep track of whom to thank, please edit the [```CONTRIBUTORS.MD```](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTORS.md) document.
 7 Have you tested your modifications by typing ```$ concepticon test``` in the command line?
-8 Have you computed the new statistics, by typing ```$ concepticon stats``` and commited the files [```concepticondata/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/README.md) and [```concepticondata/conceptlists/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/README.md)?
+8 Have you computed the new statistics, by typing ```$ concepticon stats``` and commited the files [```concepticondata/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/README.md) and [```concepticondata/conceptlists/README.md```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/README.md)? (AT: Might be deleted due to our new convention of updateing after around 10 newly added lists)
```

Comment:
Yes, @AnnikaTjuka, this does no longer belong here, as it is part of the release procedure and complicates the review process, so we need to add this in the release check list, not in the PR check list.